package.json

```diff
@@ -107,7 +107,6 @@
     "govuk-frontend": "^4.7.0",
     "helmet": "^7.0.0",
     "http-errors": "^2.0.0",
-    "jwt-decode": "^3.1.2",
```

Comment:
Removed as can't see it being used.

Reply:
Good spot 🙂! Looks like it's only used in template to get roles from HMPPS Auth token - so shouldn't be needed.